e2e-tests/README.md

```diff
@@ -0,0 +1,151 @@
+# Building and testing the Operator
+
+## Requirements
+
+You need to install a number of software packages on your system to satisfy the build dependencies for building the Operator and/or to run its automated tests:
+
+* [kubectl](https://kubernetes.io/docs/tasks/tools/) - Kubernetes command-line tool
+* [docker](https://www.docker.com/) - platform for developing, shipping, and running applications in containers
+* [sed](https://www.gnu.org/software/sed/manual/sed.html) - CLI stream editor
+* [helm](https://helm.sh/) - the package manager for Kubernetes
+* [jq](https://stedolan.github.io/jq/) - command-line JSON processor
+* [yq](https://github.com/mikefarah/yq) - command-line YAML processor
+* [krew](https://github.com/kubernetes-sigs/krew) - package manager for kubectl plugins
+* [assert](https://github.com/morningspace/kubeassert) kubectl plugin
+* [kuttl](https://kuttl.dev/) Kubernetes test framework
+* [gcloud](https://cloud.google.com/sdk/gcloud) - Google Cloud command-line tool
+
+### CentOS
```

Comment:
CentOS has been EOL lately. Maybe we should start using another Linux distro for our examples?

Reply:
It makes sense to do this switch as a separate task for this doc in all operators